<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="3" total="8" passed="5">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="42432" started-at="2014-08-29T18:49:15Z" finished-at="2014-08-29T18:49:57Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="42432" started-at="2014-08-29T18:49:15Z" finished-at="2014-08-29T18:49:57Z">
      <class name="TestCases.BatchStreamInactive">
        <test-method status="PASS" signature="SuiteLevelSetup()[pri:0, instance:TestCases.BatchStreamInactive@4e2ae]" name="SuiteLevelSetup" is-config="true" duration-ms="14156" started-at="2014-08-29T14:49:01Z" finished-at="2014-08-29T14:49:15Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SuiteLevelSetup -->
        <test-method status="PASS" signature="getDatbaseInformation()[pri:0, instance:TestCases.BatchStreamInactive@4e2ae]" name="getDatbaseInformation" is-config="true" duration-ms="3085" started-at="2014-08-29T14:49:15Z" finished-at="2014-08-29T14:49:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getDatbaseInformation -->
        <test-method status="PASS" signature="validateBatchProcesIsRunning_test01()[pri:0, instance:TestCases.BatchStreamInactive@4e2ae]" name="validateBatchProcesIsRunning_test01" duration-ms="1042" started-at="2014-08-29T14:49:18Z" finished-at="2014-08-29T14:49:19Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateBatchProcesIsRunning_test01 -->
        <test-method status="PASS" signature="seperator()[pri:0, instance:TestCases.BatchStreamInactive@4e2ae]" name="seperator" is-config="true" duration-ms="0" started-at="2014-08-29T14:49:18Z" finished-at="2014-08-29T14:49:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- seperator -->
        <test-method status="PASS" signature="getStreamInactive_test02()[pri:0, instance:TestCases.BatchStreamInactive@4e2ae]" name="getStreamInactive_test02" duration-ms="439" started-at="2014-08-29T14:49:19Z" finished-at="2014-08-29T14:49:19Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getStreamInactive_test02 -->
        <test-method status="PASS" signature="seperator()[pri:0, instance:TestCases.BatchStreamInactive@4e2ae]" name="seperator" is-config="true" duration-ms="0" started-at="2014-08-29T14:49:19Z" finished-at="2014-08-29T14:49:19Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- seperator -->
        <test-method status="PASS" signature="seperator()[pri:0, instance:TestCases.BatchStreamInactive@4e2ae]" name="seperator" is-config="true" duration-ms="0" started-at="2014-08-29T14:49:19Z" finished-at="2014-08-29T14:49:19Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- seperator -->
        <test-method status="PASS" signature="validateStreamDataCreation_test04()[pri:0, instance:TestCases.BatchStreamInactive@4e2ae]" name="validateStreamDataCreation_test04" duration-ms="1" started-at="2014-08-29T14:49:19Z" finished-at="2014-08-29T14:49:19Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateStreamDataCreation_test04 -->
        <test-method status="PASS" signature="seperator()[pri:0, instance:TestCases.BatchStreamInactive@4e2ae]" name="seperator" is-config="true" duration-ms="0" started-at="2014-08-29T14:49:19Z" finished-at="2014-08-29T14:49:19Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- seperator -->
        <test-method status="PASS" signature="setLogTime_test06()[pri:0, instance:TestCases.BatchStreamInactive@4e2ae]" name="setLogTime_test06" duration-ms="2050" started-at="2014-08-29T14:49:19Z" finished-at="2014-08-29T14:49:21Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setLogTime_test06 -->
        <test-method status="PASS" signature="setStreamInactivebatchCron_test07()[pri:0, instance:TestCases.BatchStreamInactive@4e2ae]" name="setStreamInactivebatchCron_test07" duration-ms="6477" started-at="2014-08-29T14:49:21Z" finished-at="2014-08-29T14:49:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setStreamInactivebatchCron_test07 -->
        <test-method status="PASS" signature="seperator()[pri:0, instance:TestCases.BatchStreamInactive@4e2ae]" name="seperator" is-config="true" duration-ms="0" started-at="2014-08-29T14:49:21Z" finished-at="2014-08-29T14:49:21Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- seperator -->
        <test-method status="PASS" signature="seperator()[pri:0, instance:TestCases.BatchStreamInactive@4e2ae]" name="seperator" is-config="true" duration-ms="0" started-at="2014-08-29T14:49:28Z" finished-at="2014-08-29T14:49:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- seperator -->
        <test-method status="FAIL" signature="validateStreamInactiveStart_test08()[pri:0, instance:TestCases.BatchStreamInactive@4e2ae]" name="validateStreamInactiveStart_test08" duration-ms="22066" started-at="2014-08-29T14:49:28Z" finished-at="2014-08-29T14:49:50Z">
          <exception class="java.lang.NumberFormatException">
            <message>
              <![CDATA[For input string: ""]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NumberFormatException: For input string: ""
	at java.lang.NumberFormatException.forInputString(Unknown Source)
	at java.lang.Integer.parseInt(Unknown Source)
	at java.lang.Integer.parseInt(Unknown Source)
	at SystemKeywords.Utils.convertTimeToSecond(Utils.java:13)
	at SystemKeywords.Utils.verifyLogMessage(Utils.java:43)
	at TestCases.BatchStreamInactive.validateStreamInactiveStart_test08(BatchStreamInactive.java:242)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NumberFormatException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateStreamInactiveStart_test08 -->
        <test-method status="FAIL" signature="validateStreamInactiveStop_test09()[pri:0, instance:TestCases.BatchStreamInactive@4e2ae]" name="validateStreamInactiveStop_test09" duration-ms="7029" started-at="2014-08-29T14:49:50Z" finished-at="2014-08-29T14:49:57Z">
          <exception class="java.lang.NumberFormatException">
            <message>
              <![CDATA[For input string: ""]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NumberFormatException: For input string: ""
	at java.lang.NumberFormatException.forInputString(Unknown Source)
	at java.lang.Integer.parseInt(Unknown Source)
	at java.lang.Integer.parseInt(Unknown Source)
	at SystemKeywords.Utils.convertTimeToSecond(Utils.java:13)
	at SystemKeywords.Utils.verifyLogMessage(Utils.java:43)
	at TestCases.BatchStreamInactive.validateStreamInactiveStop_test09(BatchStreamInactive.java:279)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NumberFormatException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateStreamInactiveStop_test09 -->
        <test-method status="PASS" signature="seperator()[pri:0, instance:TestCases.BatchStreamInactive@4e2ae]" name="seperator" is-config="true" duration-ms="0" started-at="2014-08-29T14:49:50Z" finished-at="2014-08-29T14:49:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- seperator -->
        <test-method status="FAIL" signature="validateStreamDataIsUpdated_test10()[pri:0, instance:TestCases.BatchStreamInactive@4e2ae]" name="validateStreamDataIsUpdated_test10" duration-ms="228" started-at="2014-08-29T14:49:57Z" finished-at="2014-08-29T14:49:57Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[STATUS not set properly in STREAM_DATA]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: STATUS not set properly in STREAM_DATA
	at org.testng.Assert.fail(Assert.java:94)
	at TestCases.BatchStreamInactive.validateStreamDataIsUpdated_test10(BatchStreamInactive.java:335)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateStreamDataIsUpdated_test10 -->
        <test-method status="PASS" signature="seperator()[pri:0, instance:TestCases.BatchStreamInactive@4e2ae]" name="seperator" is-config="true" duration-ms="0" started-at="2014-08-29T14:49:57Z" finished-at="2014-08-29T14:49:57Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- seperator -->
        <test-method status="PASS" signature="SuiteLevelTeardown()[pri:0, instance:TestCases.BatchStreamInactive@4e2ae]" name="SuiteLevelTeardown" is-config="true" duration-ms="1032" started-at="2014-08-29T14:49:57Z" finished-at="2014-08-29T14:49:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SuiteLevelTeardown -->
      </class> <!-- TestCases.BatchStreamInactive -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
